torch/_export/__init__.py

```diff
@@ -262,6 +269,8 @@ def _export(
     constraints: Optional[List[Constraint]] = None,
     *,
     strict: bool = True,
+    decomp_table: Optional[Dict[str, Callable]] = None,
+    pre_dispatch: bool = False,
```

Comment:
@tugsbayasgalan is the plan to make this default to be True in the future?

Reply:
Yep.